native/src/utils/helpers.ts

```diff
@@ -9,13 +9,75 @@ import { FetchError, NotFoundError } from 'api-client/src'
 import buildConfig from '../constants/buildConfig'
 import AppSettings from './AppSettings'
 
+const sentryEnabled = (): boolean => buildConfig().featureFlags.sentry
+const developerFriendly = (): boolean => buildConfig().featureFlags.developerFriendly
+
+export const initSentry = (): void => {
+  if (!sentryEnabled()) {
```

Comment:
Do we require mocking because of sentry? can we avoid mocking somehow?

Reply:
No, I just mock when I want to assert that the function was actually called.